asset/css/search-editor.less

```diff
@@ -18,6 +18,15 @@
     }
   }
 
+  select:not([multiple]) {
+    .appearance(none);
+    padding-right: 1.5625em;
+    background-image: url('@{iplWebAssets}/img/select-icon-white.svg');
+    background-repeat: no-repeat;
+    background-position: right center;
+    background-size: contain;
```

Comment:
`border-radius: 0` should fix the inconsistent border-radius in Safari.

![Screen Shot 2021-09-29 at 14 11 14](https://user-images.githubusercontent.com/6977434/135265962-cf61e24f-71b3-4846-8103-b04e074c0e0b.png)